util/argo/argo.go

```diff
@@ -455,3 +460,56 @@ func getDestinationServer(ctx context.Context, db db.ArgoDB, clusterName string)
 	}
 	return servers[0], nil
 }
+
+func GetAppVirtualProject(proj *argoappv1.AppProject, projLister applicationsv1.AppProjectLister, settingsManager *settings.SettingsManager) (*argoappv1.AppProject, error) {
```

Comment:
It does not look like func `GetAppVirtualProject` is exported.  Perhaps, it shouldn't be capitalized?